docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -27,14 +27,13 @@ The document below describes the database engine configurations, the necessary c
 Choosing database backend
 -------------------------
 
-If you want to take a real test drive of Airflow, you should consider setting up a database backend to **PostgreSQL**, **MySQL**, or **MSSQL**.
+If you want to take a real test drive of Airflow, you should consider setting up a database backend to **PostgreSQL**, **MySQL**.
```

Comment:
Or?

Reply:
too late :) . But yeah. It makes sense also without :)